utils/env.ts

```diff
@@ -0,0 +1,4 @@
+export const isProduction = ['kodadot.xyz', 'koda.art'].includes(
+  window.location.hostname,
+)
+export const isBeta = window.location.hostname === 'beta.kodadot.xyz'
```

Comment:
`beta.koda.art`, just in case we have that one later

Reply:
cc @hassnian

Reply:
done